Non-resolvable import POM

https://ci.eclipse.org/ejb/job/ejb-api-master-build/186/console:
```
[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: jakarta.tck:artifacts-bom:pom:11.0.2-SNAPSHOT (absent): Could not find artifact jakarta.tck:artifacts-bom:pom:11.0.2-SNAPSHOT @ line 79, column 25
```

_Originally posted by @pzygielo in https://github.com/jakartaee/enterprise-beans/issues/156#issuecomment-3137744274_

--- a/tck/pom.xml
+++ b/tck/pom.xml
@@ -79,7 +79,7 @@
             <dependency>
                 <groupId>jakarta.tck</groupId>
                 <artifactId>artifacts-bom</artifactId>
-                 <version>11.0.2-SNAPSHOT</version>
+                 <version>11.0.1-RC1</version>
                  <type>pom</type>
                  <scope>import</scope>
              </dependency>

